docs/src/main/sphinx/release/release-382.md

```diff
@@ -36,6 +36,7 @@
   reading ORC data that contains Bloom filters. ({issue}`11732`)
```

Comment:
#12506 was merged in 383. Can I ask why we should fix 382 release note?

Reply:
because I didn't know what I was doing, added https://github.com/trinodb/trino/issues/12556